runtime/compiler/control/CompilationRuntime.hpp

```diff
@@ -457,17 +457,31 @@ class CompilationInfo
    static JITServer::ServerStream *getStream();
 #endif /* defined(J9VM_OPT_JITSERVER) */
    static bool isInterpreted(J9Method *method) { return !isCompiled(method); }
-   static bool isCompiled(J9Method *method)
+
+   /**
+    * @brief Determines if a J9Method is compiled
+    *
+    * @param method pointer to the J9Method
+    *
+    * @return The start PC if compiled, NULL otherwise
+    */
+   static void* isCompiled(J9Method *method)
```

Comment:
maybe this would be better named as `getEntryIfCompiled()` or something like that?

Reply:
I could do that; do you want me to do that in this PR (noting that the PR builds are already underway and that the changes in this PR are going to be cherry-picked into the 0.23 branch) or is it ok if I make the change in another PR?

Reply:
I guess I'm thinking of it from a broader 'clean up how we interact with the extra' field approach. However, if you would want the `getEntryIfCompiled()` API in 0.23 then I might as well do it in this PR.